src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/BasicFieldsTest.java

```diff
@@ -80,4 +76,48 @@ public void test() throws ParseException {
         Assert.assertEquals("esx13.v2v.bos.redhat.com", report.getHost_name());
         Assert.assertEquals(new SimpleDateFormat("yyyy-M-dd'T'hh:mm:ss.S").parse("2019-09-18T14:52:45.871Z"), report.getCreationDate());
     }
+
+    @Test
+    public void testWithoutOptionalInputFields() throws ParseException {
+        checkLoadedRulesNumber();
```

Comment:
if you add the @before method then you can remove this line

Reply:
See above.